distributed/deploy/ssh2.py

```diff
@@ -44,6 +44,19 @@ def __await__(self):
     def __repr__(self):
         return "<SSH %s: status=%s>" % (type(self).__name__, self.status)
 
+async def _create_connection(address, validate_host):
+    if not validate_host:
+        connection = await asyncssh.connect(address, known_hosts=None)
+    else:
+        try:
+            connection = await asyncssh.connect(address)
+        except asyncssh.misc.HostKeyNotVerifiable:
+            # improve debug message when there is more support for host validation
+            logger.debug("Explicitly validating host")
+            raise asyncssh.misc.HostKeyNotVerifiable
+
+    return connection
```

Comment:
It seems like we might be able to drop this function (which would be nice to keep things simple) and optionally add the `known_hosts=None` keyword where we call `asyncssh.connect`.  I think that things would be the same but we just wouldn't get the logging, is that correct?

Reply:
correct